FE/angular/src/app/components/movie-card/movie-card.component.ts

```diff
@@ -9,4 +9,9 @@ import { MovieComplete } from 'src/app/services/data.service';
 export class MovieCardComponent {
   @Input() movie: MovieComplete = {} as MovieComplete;
   @Input() isMovieDetails = false;
+
+  onImgLoadError(event: any) {
+    event.target.src =
+      './assets/images/MV5BMGQ5YTM1NmMtYmIxYy00N2VmLWJhZTYtN2EwYTY3MWFhOTczXkEyXkFqcGdeQXVyNTA2NTI0MTY@._V1_SX300.jpg';
```

Comment:
This is not good.
You are using an existing movie image and the reasoning is not documented.
I had used a "movieNotFound.jpg' image that would have been more clear.

Reply:
Yes, I would have used a "movieNotFound.jpg' but i wanted to show the way how to handle this scenario.